tests/test_auth_tokens.py

```diff
@@ -0,0 +1,94 @@
+# pylint: disable=no-self-use
+#       - pylint test classes must pass self, even if unused.
+# pylint: disable=logging-fstring-interpolation
+#       - honestly just annoying to use lazy(%) interpolation.
+# pylint: disable=redefined-outer-name
+#       - this is how we use fixtures internally so this throws false positives
+"""
+Used to test encoding/decoding JWT tokens
+and checking for their expiration
+"""
+from datetime import timedelta
+from typing import Dict, Any
+import time
+from fastapi.testclient import TestClient
+from app import app
+from models.auth import Token
+
+client = TestClient(app)
+
+
+def create_timedelta(minutes: int, sec: int, milli: int) -> timedelta:
+    """
+    Creates timedelta in order to encode token with different times.
+    """
+    delta = timedelta(minutes=minutes, seconds=sec, milliseconds=milli)
+    return delta
+
+
+class TestAuthUser:
+    def test_encode_payload_data_ok(self,
+                                    generate_random_str_data_dict: Dict[str,
+                                                                        Any]):
+        """
+        Creates a random dict and encodes it. It then
+        makes sure it returns an encoded string.
+        """
+        payload_dict = generate_random_str_data_dict
+        encoded_token = Token.get_enc_token_str_from_dict(payload_dict)
+        assert isinstance(encoded_token, str)

```

Comment:
this assertion check is weak, we should be able to do more than just check the type. 

A good check would be to also call the `Token.check_if_valid` method, expecting the token string to be valid post-creation.